source/posts/2015-06-05-getting-started-with-ios-development.markdown

```diff
@@ -0,0 +1,102 @@
+background.  I had also previously started to learn iOS a few years ago. I spent
+a bit of time becoming familiar with Objective-C and Apple's Cocoa Touch
+frameworks.
+
+When I first got started I went searching around for some online iOS courses. I
+was hoping to find some courses on Code School or something similar.
+Unfortunately, I couldn't find any course like this. The courses that I found
+were focused around learning iOS with Objective-C. I decided if I was going to
+start learning iOS that I wanted to learn it with the modern Swift language
+rather than Objective-C.
+
+That's not to say that having Objective-C knowledge isn't valuable. It's
+especially important to have Objective-C experience to work with older
+applications. For example, [Tropos](http://troposweather.com/) is built with
+Objective-C. In order to contribute to this project, I need to think back to my
+Objective-C learnings. The specifics around iOS development tend to be the
+frameworks that Apple provides to its developers. These frameworks have great
+documentation and are documented in both, Swift and Objective-C. If you're
+learning how to use these libraries and the concepts behind iOS, it shouldn't be
+difficult to apply them to Objective-C.
+
+I was disappointed to see that there weren't many resources to get started in
+iOS from a beginner's perspective with Swift. After spending some more time
+searching I found that Stanford University has a free course on iTunes
+university titled [Developing iOS 8 Apps with
+Swift](https://itunes.apple.com/us/course/developing-ios-8-apps-swift/id961180099).
+This has been an immensely useful resource. Paul Hegarty is a great instructor
+and is really good at explaining the concepts and demonstrating things in real
+time. I highly recommend checking it out.
+
+These are a few of the important things I've learned so far:
+
+* **[Read the iOS Human Interface
+  Guidelines](https://developer.apple.com/library/ios/documentation/UserExperience/Conceptual/MobileHIG/)**
+  This may go without saying but Apple has done a ton of research behind the
+  user experience of iOS and its native applications. Fun fact that I learned:
+  The loading screen for an iOS application shouldn't be used as a splash page.
+  Instead, it should be a representation of the inactive application to give the
+  illusion of a quick load.
+
+* **Have a project idea** While reading and following the Stanford online course
+  was immensely useful to start learning, I always feel that I need a concrete
+  project to work on to apply my learnings. Without this, I usually can't keep
+  sustained interest. I recommend coming up with a simple app idea to work on to
+  build while learning. I especially got tired of the calculator application
+  example from the Stanford course.
+
+* **Read through Apple's developer documentation** While developing for iOS you
+  have a huge amount of frameworks at your disposal provided to you by Apple.
```

Comment:
change " you have a huge amount.." to " ... you have a large number of frameworks provided by Apple at your disposal." "Apple provides excellent documentation around all these frameworks; even XCode has a documentation browser built in." Add {I recommend taking the time to} "familiarize yourself with these documents and learn to navigate them, it will be a huge benefit in the long run."